src/modules/dashboard/components/DomainDropdown/DomainDropdown.tsx

```diff
@@ -0,0 +1,155 @@
+    id: 'dashboard.DomainDropdown.labelDomainFilter',
+    defaultMessage: 'Filter by Domain',
+  },
+});
+
+interface FormValues {
+  filteredDomainId: string;
+}
+
+interface Props {
+  colonyAddress: Address;
+  filteredDomainId?: number;
+}
+
+const allDomainsColor: Color = Color.Yellow;
+
+const displayName = 'dashboard.DomainDropdown';
+
+const DomainDropdown = ({ colonyAddress, filteredDomainId }: Props) => {
+  const [selectedDomain, setSelectedDomain] = useState<number>(
+    COLONY_TOTAL_BALANCE_DOMAIN_ID,
+  );
+
+  const handleSubmit = useCallback((values: FormValues) => {
+    // do stuff
+    console.log(values);
+  }, []);
+
+  const { data, error, loading } = useColonyDomainsQuery({
+    variables: { colonyAddress },
+  });
+
+  const getDomainColor = useCallback<(domainId: string | undefined) => Color>(
+    (domainId) => {
+      const defaultColor: Color = Color.Black;
+      if (domainId === '0') {
+        return allDomainsColor;
+      }
+      if (!data || !domainId) {
+        return defaultColor;
+      }
+      const domain = data.colony.domains.find(
+        ({ ethDomainId }) => Number(domainId) === ethDomainId,
+      );
+      // TODO Shouldn't have to check typeof domain.color once its return value is guaranteed via graphqlq typedefs
+      return domain && typeof domain.color === 'number'
+        ? domain.color
+        : defaultColor;
+    },
+    [data],
+  );
+
+  const renderActiveOption = useCallback<
+    (option: SelectOption | undefined, label: string) => ReactNode
+  >(
+    (option, label) => {
+      const value = option ? option.value : undefined;
+      const color = getDomainColor(value);
+      return (
+        <div className={styles.activeItem}>
+          {color && (
+            <>
+              <ColorTag color={color} />{' '}
+            </>
+          )}
+          <div className={styles.activeItemLabel}>{label}</div>
+        </div>
+      );
+    },
+    [getDomainColor],
+  );
+
+  const options = useMemo<ComponentProps<typeof Select>['options']>(() => {
+    const allDomainsOption: SelectOption = {
+      children: (
+        <DomainSelectItem
+          domain={{
+            id: '0',
+            color: allDomainsColor,
+            ethDomainId: 0,
+            name: 'All Domains',
+            ethParentDomainId: null,
+          }}
+        />
+      ),
+      label: { id: 'domain.all' },
+      value: '0',
+    };
+    if (!data) {
+      return [allDomainsOption];
+    }
+    return [
+      allDomainsOption,
+      ...data.colony.domains.map((domain) => {
+        const { ethDomainId, name } = domain;
+        return {
+          children: <DomainSelectItem domain={domain} />,
+          label: name,
+          value: `${ethDomainId}`,
+        };
+      }),
+    ];
+  }, [data]);
+
+  return (
+    <Form<FormValues>
+      initialValues={{
+        filteredDomainId: filteredDomainId ? `${filteredDomainId}` : '0',
+      }}
+      onSubmit={() => {}}
+    >
+      <Select
```

Comment:
I'm assuming the height of the dropdown is actually determined by the `Select`.

![Screenshot from 2020-11-12 17-40-40](https://user-images.githubusercontent.com/1193222/98962130-fda2b200-250e-11eb-8752-f0aa59410007.png)

Is there any way to make it taller?

Reply:
It is! Thus, I've opened an issue for this as well.